easybuild/easyconfigs/u/umap-learn/umap-learn-0.5.3-foss-2021a.eb

```
@@ -0,0 +1,44 @@
+# Author: Denis Krišťák (INUITS)
+# This is a contribution from HPCNow! (http://hpcnow.com)
+# Update: Erica Bianco <erica.bianco@hpcnow.com>
+# License::     GPL-v3.0
+easyblock = "PythonBundle"
+
+name = 'umap-learn'
+version = '0.5.3'
+
+homepage = 'https://umap-learn.readthedocs.io/en/latest/'
+description = """Uniform Manifold Approximation and Projection (UMAP) is a dimension reduction technique 
+                 that can be used for visualisation similarly to t-SNE, but also for general non-linear 
+                 dimension reduction."""
+
+toolchain = {'name': 'foss', 'version': '2021a'}
+
+
+dependencies = [
+    ('Python', '3.9.5'),
+    ('SciPy-bundle', '2021.05'),
+    ('scikit-learn', '0.24.2'),
+    ('numba', '0.53.1'),
+    ('LLVM', '11.1.0'),
+    ('tqdm', '4.61.2'),
+]
+
+use_pip = True
+download_dep_fail = True
+
+exts_default_options = {'source_urls': [PYPI_SOURCE]}
+
+exts_list = [
+    ('pynndescent', '0.5.7', {
+        'checksums': ['ecb395255fa36a748b5870b4ba0300ea0f7da8b1964864b8edd62577a84dfd7d'],
+    }),
+    (name, version, {
+        'modulename': 'umap',
+        'checksums': ['dbd57cb181c2b66d238acb5635697526bf24c798082daed0cf9b87f6a3a6c0c7'],
+    }),
+]
+
+sanity_pip_check = True
+
+moduleclass = 'lib'
```

for consistency with existing easyconfigs for `umap-learn`, we should stick to the `tools` moduleclass
```suggestion
moduleclass = 'tools'
```